donut/modules/calendar/static/script.js

```diff
@@ -0,0 +1,388 @@
+var CURRENT_DATE = new Date();
+var d = new Date();
+
+var content = 'January February March April May June July August September October November December'.split(' ');
+var weekDayName = 'SUN MON TUES WED THURS FRI'.split(' ');
+var daysOfMonth = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
```

Comment:
Should use `Date` objects to get this information. For example `new Date(2019, 5, 0).getDate()` will give you the number of days in May 2019. Date math is notoriously hard to get right; for example, February doesn't always have 28 days.